config/templates/stack-template.json

```diff
@@ -509,7 +509,7 @@
     {{range $etcdIndex, $etcdInstance := .EtcdInstances}}
     "InstanceEtcd{{$etcdIndex}}eni": {
       "Properties": {
-          "SubnetId": {{$etcdInstance.SubnetRef}},
+          "SubnetId": {{$etcdInstance.Subnet.Ref}},
```

Comment:
Thanks for suggesting it 👍 
However I'd prefer to keep this as before while changing `$etcdInstance.Subnet.AvailabilityZone` to `$etcdInstance.SubnetAvailabilityZone` so that we don't need to expose everything related to a `Subnet` to templates.
`SubnetRef` and `SubnetAvailabilityZone` are the only things required in stack templates, aren't they?
Could I have your thoughts?